lib/active_fedora/relation/finder_methods.rb

```diff
@@ -105,12 +105,18 @@ def exists?(conditions)
     # @param[Hash] options
     # @option opts [Array] :sort a list of fields to sort by
     # @option opts [Array] :rows number of rows to return
-    def find_with_conditions(conditions, opts = {})
+    def search_with_conditions(conditions, opts = {})
       # set default sort to created date ascending
       opts[:sort] = @klass.default_sort_params unless opts.include?(:sort)
       SolrService.query(create_query(conditions), opts)
     end
 
+    # @deprecated
+    def find_with_conditions(*args)
+      Deprecation.warn(ActiveFedora::Base, '.find_with_conditions is deprecated; use .search_with_conditions instead')
```

Comment:
Should we add that it'll be going away in 11.0.0 or some such indication of the *when*?